src/main/java/com/mazyavr/schedule/controller/GoogleController.java

```diff
@@ -86,11 +96,19 @@ public RedirectView googleCallback(
         return new RedirectView(redirectUri);
     }
 
+    @Operation(
+        summary = "Авторизованный ответ",
+        description = "Токен не должен быть нулевым"
+    )
     @GetMapping(value = "/is-authorized")
     public @ResponseBody IsAuthorizedResponse isAuthorized() {
         return new IsAuthorizedResponse(token != null);
     }
 
+    @Operation(
+        summary = "Получение событий из google-календаря",
+        description = "Позволяет загрузить событие из google-календаря"
```

Comment:
В description написано по сути то же самое что в summary, так что последнее не нужно. Меньше воды больше смысла

```suggestion
        summary = "Получение событий из google-календаря"
```